modules/apps/frontend-js/frontend-js-spa-web/src/main/resources/META-INF/resources/senna/app/App.js

```diff
@@ -1509,7 +1511,7 @@ class App extends EventEmitter {
 		};
 
 		return new CancellablePromise(
-			(resolve) => sync() & async.nextTick(() => sync() & resolve())
+			(resolve) => sync() & setTimeout(() => sync() & resolve(), 0)
```

Comment:
The `&` operator (bitwise `AND`) is rather opaque here... I remember looking into this a while back but I've forgotten what it was for... do we need it?

Reply:
I honestly don't know... but we'll be removing `CancellablePromise` next, so we'll double back on this soon enough

Reply:
https://github.com/liferay-frontend/liferay-portal/pull/489#issuecomment-725325366